### ISSUE TITLE
Chord startChar

### ISSUE THREAD

**Issue by ehsansh** [paulrosen/abcjs#663]
If I click on this note `"^V"TG` then in the click event by this line of code
```js
const partOfOriginalString = tune.slice(abcElem.startChar, abcElem.endChar);
console.log( partOfOriginalString )
```
I could get the exact string that represents the note. 
But If I click on a chord like this `"^V"T[G1B1D1]`
The start character starts from `[` and I get this string: `[G1B1D1]`
I thought `startChar` starts from any character that the note starts. but it doesn't. Why?
Should I calculate `chord` and `decoration` values to find the exact string of the chord?

**Comment by paulrosen** [comment]
I think that is a regression error. I'll look into it. It should be the whole thing, and it used to be.

**Comment by paulrosen** [comment]
Fixed in 6.0.0-beta.33
